pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <groupId>org.openmicroscopy</groupId>
   <artifactId>ome-model</artifactId>
-  <version>6.0.0-SNAPSHOT</version>
+  <version>5.5.0-SNAPSHOT</version>
```

Comment:
As this is a multi-module component, the versions in all the POMs should be updated